catalog-info.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: py-sqs-queue
+  description: |
+    Simple Python AWS SQS queue consumer and publisher.
+  tags:
+    - python
+    - sqs
+  annotations:
+    github.com/project-slug: Media-Platforms/py-sqs-queue
```

Comment:
Could we annotate which ones we've open-sourced?

Maybe add their PyPI URLs...

Reply:
Added in links as there is no "official" pypi annotation in the docs.